apps/web/moon.yml

```diff
@@ -0,0 +1,112 @@
+language: 'javascript'
+
+fileGroups:
+  tests:
+    - 'tests/**/*'
+    - '**/*.test.ts*'
+  lint:
+    - '**/*.*{ts,js,tsx,jsx,cjs,mjs}'
+  source:
+    - 'src/**/*.{ts,js,tsx,jsx}'
+    - 'pages/**/*.{ts,js,tsx,jsx}'
+
+tasks:
+  build:
+    deps:
+      - '~:relay-codegen'
+      - '~:graphql-codegen'
+    command: next build
+    inputs:
+      - '@group(source)'
+      - 'next.config.js'
+
+  cypress-gh:
+    command:
+    - synpress run --record
+    - --key 9d633f22-7630-4a11-9cf4-ff9809332564
+    - --configFile cypress.config.ts
+
+  dev:
+    command:
+    - concurrently
+    - -n Next,Relay-Compiler,GraphQL-Codegen
+    - yarn next dev
+    - yarn moon run web:relay-watch
+    - yarn moon run web:graphql-codegen-watch
+
+  lint:
+    command:
+    - eslint
+    - --ext
+    - .js,.ts,.tsx,.cjs
+    - '@group(lint)'
+    inputs:
+      - '@group(lint)'
```

Comment:
normal to have the same command and inputs?

Reply:
I'm pretty sure this is normal? cc @milesj

Reply:
Yeah, they just get expanded.